frontend/src/scenes/onboarding/Onboarding.tsx

```diff
@@ -134,6 +134,14 @@ const ProductAnalyticsOnboarding = (): JSX.Element => {
             type: 'toggle',
             visible: true,
         },
+        {
+            title: 'Enable web vitals autocapture',
+            description: `Uses Google's web vitals library to automagically capture performance information.`,
+            teamProperty: 'autocapture_web_vitals_opt_in',
+            value: currentTeam?.autocapture_web_vitals_opt_in ?? true,
+            type: 'toggle',
+            visible: true,
+        },
```

Comment:
Do we want to add this to replay, flags, and surveys as well? So people can turn it on regardless of what product they are using.

Reply:
🤔 feels weird to me to have cross-product toggles... but then we have session replay in the product analytics one

i might leave things as they are for now